functions/go/create-setters/README.md

```diff
@@ -0,0 +1,135 @@
+# create-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Create Setters are the template-free solution to set a comment for matching resource fields.
```

Comment:
Add setter comments to matching resource fields. Setters serve as parameters for template-free setting of field values.